api/access/server_test.go

```diff
@@ -120,13 +121,14 @@ func TestServer_GetBlockHeaderByID(t *testing.T) {
 	t.Run("handles indexer error on HeightForBlock", func(t *testing.T) {
 		index := mocks.BaselineReader(t)
 		index.HeightForBlockFunc = func(blockID flow.Identifier) (uint64, error) {
-			return 0, mocks.DummyError
+			return 0, mocks.GenericError
 		}
 
 		s := baselineServer(t)
 		s.index = index
 
-		req := &access.GetBlockHeaderByIDRequest{Id: mocks.GenericIdentifiers[0][:]}
+		id := mocks.GenericIdentifier(0)
+		req := &access.GetBlockHeaderByIDRequest{Id: id[:]}
```

Comment:
If this annoys you, you can do a wrapper function, like:

```go
mocks.IdentifierBytes(mocks.GenericIdentifier(0))
```

It's a little more explicit, but I don't mind either way.

Reply:
I think it's easier this way, the `mocks` package already feels quite cluttered with all of our generic values so I'd like to keep it as minimalistic as possible